packages/sales-api-service/src/services/renewals/renewals.service.js

```diff
@@ -83,4 +90,20 @@ const dateDataIfNotExpired = endDateMoment => {
   }
 }
 
-const preparePermitId = existingPermission => existingPermission.permit.id
+const preparePermit = async existingPermission => {
+  const permit = await findPermit(existingPermission)
+  return permit.id
+}
+
+const prepareConcessionsData = async (existingPermission, dateData) => {
+  delete existingPermission.licensee.noLicenceRequired
+  const ageAtLicenceStartDate = moment(dateData.licenceStartDate)
+    .add(1, 'year')
+    .diff(moment(existingPermission.licensee.birthDate), 'years')
+
+  if (isSenior(ageAtLicenceStartDate)) {
+    await addSenior(existingPermission)
+  }
+
+  return existingPermission
```

Comment:
Is there a reason to not just return `existingPermission.concessions` given that that's the only data we are using from it? Would make `preparePermissionDataForRenewal()` a bit less complex too.